python/fava/Portfile

```diff
@@ -23,7 +23,7 @@ homepage                https://beancount.github.io/fava/
 master_sites            pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname                ${python.rootname}-${version}
```

Comment:
`master_sites` and `distname` can be removed, these are the defaults set by the `python` PG